src/borg/archiver.py

```diff
@@ -2147,7 +2154,8 @@ def do_break_lock(self, args, repository):
             path. The first matching pattern is used so if an include pattern matches before
             an exclude pattern, the file is backed up. If an exclude-norecurse pattern matches
             a directory, it won't recurse into it and won't discover any potential matches for
-            include rules below that directory.
+            include rules below that directory. Note that this is the behavior that gets applied
```

Comment:
I thought about whether this should be mentioned higher up in the section about `--exclude`, but I think it makes more sense here, since the difference in behavior is only relevant if you are using the experimental `--pattern` selectors in this section.